packages/eslint-plugin/src/utils/configHelpers.js

```diff
@@ -84,7 +85,7 @@ module.exports = {
    *   - may need to reconsider for converged components depending on website approach
    *   - the stories suffix is also used for storywright stories in `vr-tests`
    */
-  devDependenciesFiles: [...testFiles, ...docsFiles, ...configFiles, '**/*.stories.tsx'],
+  devDependenciesFiles: [...testFiles, ...docsFiles, ...configFiles, ...storyFiles],
```

Comment:
react-virtualizer doesn't follow other packages patterns -> using `index.stories.tsx`. This was also reason why those files were incorrectly linted agains extrenaous dependencies rule.  this fixes this incorrect DX

cc @Mitch-At-Work